.github/workflows/config/mlc_config.json

```diff
@@ -2,35 +2,53 @@
     "ignorePatterns": [
         {
             "pattern": "https://github.com/Your_Github_Handle/owasp-mstg"
```

Comment:
this one and the 3rd one are present in some .md file, but the second one ... I cannot tell why it's there. Once everything with the headers, reties, exclusions etc is working fine we should try to remove this and check if there are some errors in the log.